src/containers/pages/MDAPoint/ChildReports/index.tsx

```diff
@@ -68,6 +68,27 @@ const tableHeaders = [
   ALB_TABLETS_DISTRIBUTED,
 ];
 
+/*
+ * returns list of childrens
+ * @param {ChildReport[]} props
+ * @return - extracted list of child data
+ */
+export const extractChildData = (data: ChildReport[]) => {
+  return data.map((sch: ChildReport) => {
+    return [
+      `${sch.client_first_name} ${sch.client_last_name}`,
+      sch.sactanationalid ? sch.sactanationalid : '',
+      sch.sactacurrenroll ? YES : NO,
+      sch.mmadrugadmin ? YES : NO,
+      sch.mmanodrugadminreason ? YES : NO,
+      sch.mmanodrugadminreason ? YES : NO,
```

Comment:
Let's confirm if we need to use same data for two columns. I.e `SACs Refused` and `SACs Sick/Pregnant/Contraindicated`

Reply:
@isabelcshaw @samkanga

Reply:
@baokiddy @gstuder-ona